Sync `css/css-font-loading` from WPT upstream

Ahmad-S792 opened the pull request:
#### a8d4b40470142b956f1f9e2185482f61016c6485
<pre>
Sync `css/css-font-loading` from WPT upstream
<a href="https://bugs.webkit.org/show_bug.cgi?id=295293">https://bugs.webkit.org/show_bug.cgi?id=295293</a>
<a href="https://rdar.apple.com/154781496">rdar://154781496</a>

Reviewed by Vitor Roriz.

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/2c26d7d84ecb5b12d4ee2a567a1a66290b01bc6d">https://github.com/web-platform-tests/wpt/commit/2c26d7d84ecb5b12d4ee2a567a1a66290b01bc6d</a>

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-descriptor-updates-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-override-descriptors-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-size-adjust-descriptor-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-clear-css-connected-2-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-delete-css-connected-2-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-has.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-loading-worker-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-worker-fontface-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/support/fontfaceset-loading-worker.js:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/w3c-import.log:
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords-expected.txt:

Canonical link: <a href="https://commits.webkit.org/296889@main">https://commits.webkit.org/296889@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/cc4eb6cfc06a814c7e39b98e9911e12647c033d7

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/109873 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/29531 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/19961 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/115894 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/60110 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/30209 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/38119 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/83528 "Passed tests") | [⏳ 🧪 win-tests](https://ews-build.webkit.org/#/builders/Win-Tests-EWS "Waiting to run tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/112821 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/24084 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/98947 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/63969 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/23458 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/17095 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/59689 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/93455 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/17137 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/118686 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/36912 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/27356 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/92506 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/37285 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/95213 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/92329 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23523 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/37305 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/15044 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/32758 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/36807 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/36467 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/39809 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/38176 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

vitorroriz reviewed (approved):
I think we should update the expected results on WPE for fontfaceset-load-css-wide-keywords

Ahmad-S792 committed:
Sync `css/css-font-loading` from WPT upstream
https://bugs.webkit.org/show_bug.cgi?id=295293
rdar://154781496

Reviewed by Vitor Roriz.

Upstream commit: https://github.com/web-platform-tests/wpt/commit/2c26d7d84ecb5b12d4ee2a567a1a66290b01bc6d

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/WEB_FEATURES.yml:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-descriptor-updates-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-override-descriptors-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontface-size-adjust-descriptor-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-clear-css-connected-2-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-delete-css-connected-2-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-has.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-loading-worker-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-worker-fontface-crash.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/support/fontfaceset-loading-worker.js:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/support/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-font-loading/w3c-import.log:
* LayoutTests/tests-options.json:
* LayoutTests/platform/glib/imported/w3c/web-platform-tests/css/css-font-loading/fontfaceset-load-css-wide-keywords-expected.txt:

Canonical link: https://commits.webkit.org/296889@main
